specification/advisor/resource-manager/readme.md

```diff
@@ -26,7 +26,31 @@ These are the global settings for the Advisor API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2025-05-preview
+tag: package-2025-08-preview
+```
+
+### Tag: package-2025-08-preview
+These settings apply only when `--tag=package-2025-08-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2025-08-preview'
+input-file:
+  - Microsoft.Advisor/preview/2025-08-01-preview/advisor.json
+  - Microsoft.Advisor/preview/2025-08-01-preview/assessments.json
+  - Microsoft.Advisor/preview/2025-08-01-preview/resiliencyReviews.json
+suppressions:
+  - code: ArmResourcePropertiesBag
+    reason: Suppressing for now to avoid breaking change.
```

Comment:
>Suppressing for now to avoid breaking change. [](http://example.com/codeflow?start=12&length=45)

if this is temporary supprestion, what is your long term plan to address it. Suppressing it only continue to create confusion for customers and violates resource modelling principal. If this is going to be addressed in GA version, suggestion is to better address it now.